cg/services/orders/order_service/models.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import BaseModel, field_validator, Field
+from cg.constants import Workflow
+from cg.server.dto.orders.orders_request import OrderSortField, SortOrder
+
+
+class OrderQueryParams(BaseModel):
+    page_size: int | None = Field(default=50)
+    page: int | None = Field(default=1)
+    sort_field: str | None = Field(default=OrderSortField.ORDER_DATE)
+    sort_order: str | None = Field(default=SortOrder.ASC)
+    search: str | None = None
+    workflows: list[str] | None = []
+    is_open: bool | None = None
+
+    @field_validator("workflows", mode="before")
+    def expand_balsamic_workflow(cls, value):
```

Comment:
I see @islean now, implementing this in the OrdersRequest class would be much simpler. Here we are adding a new model with exactly the same attributes but with an extra validation. I also understand @ChrOertlin concern on decoupling layers. Decoupling makes changes easier but maybe we doon't want to incorporate changes here that often? This is an interesting design pattern that already has been considered in the order flow project and we could discuss better